clang/include/clang/Sema/Sema.h

```diff
@@ -5432,6 +5432,9 @@ class Sema final : public SemaBase {
   void MarkBaseAndMemberDestructorsReferenced(SourceLocation Loc,
                                               CXXRecordDecl *Record);
 
+  void MarkBaseDestructorsReferenced(SourceLocation Loc, CXXRecordDecl *Record);
+  void MarkFieldDestructorReferenced(SourceLocation Loc, FieldDecl *Field);
```

Comment:
I feel these should just be static functions in the cpp file

Reply:
done